src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this.bindEvents();
+    }
+
+    /**
+     * Sets entered number on the screen.
+     */
+    bindEvents() {
+        let buttons = document.querySelector('.allButtons');
+        let numberButtons = buttons.querySelectorAll('.number');
+
+        for (let i = 0; i < numberButtons.length; i++) {
+            let self = this;
+            numberButtons[i].onclick = function() {
```

Comment:
Using this method (attaching a function directly to `.onclick`) you can register only one handler. But if you use `addEventListener` for this event, you can attach multiple functions (due to event based approach, same like we are doing for DigitKeyboard).

The general rule of thumb is to always use attaching to an event. Please update the code.